cli/index.js

```diff
@@ -39,13 +43,354 @@ const signUp = (args) => {
 
             const FBIdToken = `Bearer ${res.data.token}`;
             // TODO: create a file in the user's machine to store the auth token
-            console.log(chalk.white.bold("Account created successfully ✔️"));
+            console.log(chalk.green.bold("Account created successfully ✔️"));
         })
         .catch((error) => {
             console.log(chalk.red.bold(`Could not create account -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
         });
 }
 
+// *definition of create guest command
+const guestSignUp= (args) => {
+
+    const { picture, username } = args;
+
+    const guestuserdata= {
+        picture,
+        username,
+    };
+
+    console.log(chalk.yellow.bold("Creating a guest account..."));
+
+    axios
+        .post("/user/guest", guestuserdata)
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            // TODO: create a file in the user's machine to store the auth token
+            console.log(chalk.green.bold("user guest created successfully ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not create aguest account -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+
+//* Login 
+const login = (args) => {
+
+    const { password, username } = args;
+
+    const useUserData = {
+        password,
+        username,
+    };
+
+    console.log(chalk.yellow.bold("login..."));
+
+    axios
+        .post("/user/login", useUserData)
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            // TODO: create a file in the user's machine to store the auth token
+            console.log(chalk.green.bold("login successfully ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not login -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+
+// implement user/logout
+   {  console.log(chalk.yellow.bold("Logging out..."));
+
+    axios
+        .post("/user/logout")
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            
+            console.log(chalk.green.bold(" successfully operation ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not log out: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+
+    }
+//* recover 
+const recover = (args) => {
+
+    const { username, userId, email, newPassword, confirmPassword } = args;
+
+    const recoverUserData = {
+        username,
+        userId,
+        email,
+        newPassword,
+        confirmPassword,
+    };

```

Comment:
the same thing again, it doesn't make sense to update the userId and email when recovering the account. We only want to update the password